src/plone/rest/testing.py

```diff
@@ -14,6 +13,7 @@ class PloneRestLayer(PloneSandboxLayer):
     defaultBases = (PLONE_APP_CONTENTTYPES_FIXTURE,)
 
     def setUpZope(self, app, configurationContext):
+        z2.installProduct(app, 'plone.app.theming')
```

Comment:
@displacedaussie could you please elaborate why this is necessary here? I would like to keep the plone.rest test fixture as lightweight as possible.

Reply:
Otherwise portal_resources doesn't exist. And that is what plone.rest is messing with the traversal of.